pyslackers_website/slack/templates/slack/partials/member_map.html

```diff
@@ -0,0 +1,19 @@
+{% load static %}
+
+<div class="ui container" >
+  <div class="ui container segment" id="leaflet_container">
+    <div id="leaflet_member_map" />
+  </div>
+</div>
+<script type="text/javascript">
+  const slack_member_tz_count = {{ slack_member_tz_count | safe }};
```

Comment:
I get a bit nervous depending on a global variable to exist for a script to be able to run properly - but I don't have any great ideas other than making it an arg to an IIFE (As commented above).

I do need to validate what happens when a partial does these tags and the main template this is included into defines its own, I didn't think about that prior to recommending it. In jinja you have the `super()` call, django I don't think does.

Reply:
The code itself won't get included if we don't have this variable defined, does that fit the required constraint?